LayoutTests/accessibility/roles-exposed.html

```diff
@@ -218,6 +218,8 @@ <h2 id="section-label">X</h2>
 <ul data-platform="atspi,mac" class="ex">
     <li data-platform="atspi,mac" class="ex">X</li>
 </ul>
+
+
```

Comment:
Why this whitespace change? I guess you mean to add a test for menu here as suggested by @twilco?

Reply:
Yes. I must have left this in.